RELEASE.md

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
the release is 1.2.0
it is wrong in develop.
I already talked with Jakob about it :)